template/INSTRUCTIONS.md.tpl

```diff
@@ -0,0 +1,7 @@
+# Instructions
```

Comment:
I might consider renaming this `READ_BEFORE_COMMIT.md` or something a little more explicit about it being used before submitting the first PR.